src/community/support.md

```diff
@@ -21,4 +21,4 @@
   + [Sign up for Office Hours!](https://docs.google.com/forms/d/e/1FAIpQLSeMhZtzQQWB47I8HfPcJ5_pFyMhZO284PLIblDfshe30dEuXw/viewform)
 
 For fully managed services, we invite you to visit 
-[DataJoint](https://www.datajoint.com) for solutions and request more information.
+[DataJoint](https://www.datajoint.com) for solutions or to [chat with our teeam](./#hs-chat-open) for more information.
```

Comment:
Hi @mkosma - Where are you trying to direct to? The `./#notation` directs to a heading within a page (example [webpage](https://datajoint.com/docs/core/concepts/mantra/) and [corresponding markdown](https://raw.githubusercontent.com/datajoint-company/datajoint-docs/main/src/core/concepts/mantra.md))

Reply:
Indeed. That nomenclature opens the chat widget on the current page.  See, e.g., https://community.hubspot.com/t5/Tips-Tricks-Best-Practices/How-to-Automatically-open-the-chat-widget-when-a-customer-loads/m-p/378386